src/br/com/cursosLab/model/Course.kt

```diff
@@ -0,0 +1,51 @@
+package br.com.cursosLab.model
+
+import java.lang.Exception
+
+class Course(
+    val name: String,
+    val id: String ,
+    val time: Int ,
+    val visibility: Boolean = false,
+    val targetPublic: String = "",
+    val instructor: String ,
+    val pathway: String = "",
+    val skill: String = "",
+) {
+    init {
+        nameVerify()
+        idVerify()
+        instructorVerify()
+        timeVerify()
+
+    }
+
+    private fun nameVerify() {
+        val regex = "^[A-z]+\$".toRegex()
```

Comment:
as you're using this regex twice in your code, i recomend you to create a const and put there this value!